bikeshed/markdown.py

```diff
@@ -337,6 +337,8 @@ def parseParagraph(stream):
 	elif line.lower().startswith("issue:"):
 		line = line[6:]
 		p = "<p class='replace-with-issue-class'>"
+        elif line.lower().startswith("assert:"):
```

Comment:
Huh. Actually, no, I write my Python with spaces per PEP8, but this file must have slipped tabs in without me realizing. I'll make sure everything is consistent in a bit. ^_^